app/views/govuk_publishing_components/components/_service_navigation.html.erb

```diff
@@ -0,0 +1,46 @@
+<%
+  add_gem_component_stylesheet("service-navigation")
+
+  service_name ||= nil
+  navigation_items ||= []
+
+  component_helper = GovukPublishingComponents::Presenters::ComponentWrapperHelper.new(local_assigns)
+  component_helper.add_class("gem-c-service-navigation govuk-service-navigation")
+  component_helper.add_aria_attribute({ label: "Service information" })
+  component_helper.add_data_attribute({ module: "govuk-service-navigation" })
+%>
+<% if navigation_items.present? %>
+  <%= tag.section(**component_helper.all_attributes) do %>
+    <div class="govuk-width-container">
+      <div class="govuk-service-navigation__container">
+        <% if service_name %>
+          <span class="govuk-service-navigation__service-name">
+            <a href="#" class="govuk-service-navigation__link">
+              <%= service_name %>
+            </a>
+          </span>
+        <% end %>
+
+        <% if navigation_items.present? %>
+          <nav aria-label="<%= t("components.layout_header.menu") %>" class="govuk-service-navigation__wrapper">
+            <button type="button" class="govuk-service-navigation__toggle govuk-js-service-navigation-toggle" aria-controls="navigation" hidden>
+              Menu
+            </button>
+            <ul class="govuk-service-navigation__list" id="navigation">
+              <% navigation_items.each do |nav| %>
+                <% 
+                  nav_classes = %w(govuk-service-navigation__item)
+                  nav_classes << "govuk-service-navigation__item--active" if nav[:active]
+                  aria_current = nav[:active]
+                %>
+                <%= tag.li nav[:text], class: nav_classes do %>
+                  <%= link_to nav[:text], nav[:href], class: "govuk-service-navigation__link", aria: { current: aria_current } %>
```

Comment:
After looking at the [example in the design system](https://github.com/alphagov/govuk-frontend/blob/5786234c2fc2e89e2b6a925211ca1cfd38fda146/packages/govuk-frontend/src/govuk/components/service-navigation/template.njk#L47-L56) we may need to update the markup here for active links, for example they also wrap active links in a `strong` tag for the reasons below:

> We wrap active links in strong tags so that users who override colours or styles will still have some indicator of the current nav item.

Reply:
Thanks, good spot. Now updated